src/nupic/utils/LoggingException.cpp

```diff
@@ -34,7 +34,11 @@ LoggingException::~LoggingException() throw()
   if (!alreadyLogged_) {
     // Let LogItem do the work for us. This code is a bit complex
     // because LogItem was designed to be used from a logging macro
+#if defined(NTA_OS_WINDOWS) && defined(NTA_COMPILER_MSVC) && defined(NDEBUG)
+    auto li = new LogItem(filename_.c_str(), lineno_, LogItem::info);
```

Comment:
OK so I looked into the code slightly more. This is an exception class so it makes sense that it is specifying `error` instead of `info`. I think it would be logically incorrect to switch it to `info`, even if it is just for Windows. So I see two scenarios:

1. This is being called somewhere when there isn't actually an error. In this case, we should change the calling code to not call this unless there is actually an error.
2. There is actually an error but we don't want the AppVeyor build to fail. In this case, we should look at what we are running in AppVeyor that leads to the failure and see if it makes sense to suppress the error output or something.

Can one of you point me to the output from the AppVeyor failure?